winch/codegen/src/isa/x64/mod.rs

```diff
@@ -94,4 +94,32 @@ impl TargetIsa for X64 {
 
         Ok(masm.finalize())
     }
+
+    fn compile_trampoline(
```

Comment:
I've been testing locally, but both of these should be filled out for `x64` so our first pass works on both ISAs